src/index.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const App = express();
```

Comment:
O nome da constante deveria ser iniciado com letra minúscula
```suggestion
const app = express();
```